src/main/java/pages/yandex/MailPage.java

```diff
@@ -23,5 +24,8 @@ public MailPage switchToLightVersion(){
         driver.findElement(LIGHT_VERSION_BUTTON).click();
         return new MailPage(driver);
     }
-
+    public MailPage logOut(){
```

Comment:
You can't be on the same page after logout

Reply:
?

Reply:
Agree, I created new class YandexMainPage